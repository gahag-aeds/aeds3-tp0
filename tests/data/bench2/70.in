70 35
17 77 19 71 66 17 88 59 71 47 52 49 42 39 24 10 66 25 56 67 67 12 22 57 71 86 52 93 94 11 64 11 40 35 83 6 52 71 65 23 70 69 24 12 9 1 75 75 26 31 43 45 95 17 2 19 3 54 12 98 17 28 9 9 63 44 15 67 67 33 
Add 9 31
Min 42 64
Add 33 55
Min 25 47
Add 40 62
Min 20 42
Sub 20 42
Max 17 39
Sub 32 54
Min 1 23
Add 20 42
Max 19 41
Add 5 27
Sum 3 25
Sub 37 59
Sum 29 51
Sub 34 56
Sum 40 62
Add 8 30
Min 35 57
Add 46 68
Max 18 40
Sub 13 35
Max 2 24
Add 38 60
Sum 2 24
Add 23 45
Max 41 63
Sub 41 63
Max 30 52
Sub 39 61
Sum 28 50
Sub 3 25
Min 24 46
Sub 5 27
