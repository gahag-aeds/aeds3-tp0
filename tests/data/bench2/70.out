1
2
7
94
11
95
1135
989
835
0
96
89
1144
101
82
1032
0
